### PR TITLE
Adds task to generate user objects from a csv (tab sepatated)

### LINKED ISSUE #1298: Add rake task to generate new user accounts
Can be done after the beta preview.

Related work in 2.x: https://github.com/uclibs/scholar_uc/commit/e86e2f7fa7ec61de368fbcbed3aa9b90ea02f8d7

### PR DESCRIPTION
**Pull request by 13miciunasl**
Fixes #1298 ; refs #1572 

Adds a rake task that generates users from a csv tab separated file

Use: rake account:generate["your_file_path_here"]

### CODE REVIEW

**Comment by jamesvanmil** [comment]
@13miciunasl can you add a comment to the rake task with sample syntax for running the command? also can you please email me sample data for running the command locally?

File: lib/tasks/account.rake [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    file = CSV.read(args[:file_path], headers: true, col_sep: "\t", quote_char: "\x00")
+    file.each do |user|
+      if user["sixplus2"].to_s.present?
+        if generate_uc_account(user)
+          puts "Account generated for #{user['first_name']} #{user['last_name']} (#{user['email']})"
+        else
+          puts "Account could not be generated. A user with email #{user['email']} may already exist."
+        end
+      else
+        puts "Usage:   rake account:generate['sixplus2','Firstname','Lastname'] (no spaces between the parameters)"
```
Comment (jamesvanmil) [inline lib/tasks/account.rake]: Is this usage still valid, or does it always read from file now?

Comment (13miciunasl) [reply lib/tasks/account.rake]: It is not! Good call. I'll change that

**Comment by jamesvanmil** [comment]
I sketched out a quick refactor to remove the long definition of the attributes hash:

```ruby
module AccountHelper
  def generate_uc_account(attributes)
    unless User.where(email: attributes["email"]).exists?
      attributes = attributes.to_hash
      proxy_emails = attributes.delete(:proxy_emails)
      attributes[:password] = (0...16).map { ('a'..'z').to_a[rand(26)]  }.join
      new_user = User.create attributes
      add_proxy_users(new_user, proxy_emails) 
    end
  end

  private

    def add_proxy_users(user, proxy_emails)
      return if proxy_emails.nil?
      proxy_emails.split("|").each do |email|
        user.can_receive_deposits_from << User.where(email: email).first
      end
      user.save
    end
end

```

**Comment by jamesvanmil** [comment]
Also - the sixplus2 var isn't actually doing anything?

**Comment by jamesvanmil** [comment]
@13miciunasl everything seems to check out - we're going to talk about these brakeman failures today

**Pull request merged**